frontend/components/metric-progress-circular.vue

```diff
@@ -10,6 +10,8 @@
     <div :class="labelClass">
       <h3 class="font-weight-medium">{{ label }}</h3>
       <span v-if="typeof value === 'number'">{{value.toFixed(1)}}%</span>
+      <span v-else-if="typeof value === null">null</span>
```

Comment:
As above, I don't think it's important to tell the user "null" vs. "undefined", but instead tell them what could be wrong with their data. In our case, there's not different between null and undefiend